src/tools/ReadyToRun.SuperIlc/PathHelpers.cs

```diff
@@ -247,24 +248,30 @@ private static async Task LocateOutputFoldersAsync(string folder, string coreRoo
             {
                 if (Path.GetExtension(dir).Equals(".out", StringComparison.OrdinalIgnoreCase))
                 {
-                    directories.Add(dir);
+                    foreach (CompilerRunner runner in runners)
```

Comment:
Nit - why cannot we just proactively mark GetOutputPath for each [existing] runner for deletion (let's say assuming it exists)?